ios/Magnet.xcodeproj/project.pbxproj

```diff
@@ -529,7 +552,7 @@
 			isa = PBXProject;
 			attributes = {
 				LastUpgradeCheck = 0610;
-				ORGANIZATIONNAME = Facebook;
+				ORGANIZATIONNAME = Mozilla;
```

Comment:
I'm sure we changed this already, but my files kept saying: "Copyright Facebook"